runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingSideInputFetcher.java

```diff
@@ -87,6 +89,9 @@ public StreamingSideInputFetcher(
             StateTags.bag("elem", WindowedValue.getFullCoder(inputCoder, mainWindowCoder)));
     this.timersAddr =
         StateTags.makeSystemTagInternal(StateTags.bag("timer", TimerDataCoder.of(mainWindowCoder)));
+    this.oldTimersAddr =
+        StateTags.makeSystemTagInternal(
+            StateTags.bag("oldTimer", OldTimerDataCoder.of(mainWindowCoder)));
```

Comment:
This is reversed - we want oldTimersAddr to match data in existing pipelines for compatibility. This means that oldTimersAddr must be what was there before.

Reply:
@reuvenlax , done